docs/variants/clevo_nv41/lvfs_report.md

```diff
@@ -25,64 +25,64 @@ NovaCustom NV4x system firmware
 ### Feasibility analysis
```

Comment:
I wonder why we don't use markdown notation for links in this section. Also we haven't ever used `<>` to encapsulate links. To be consistent I would make markdown links

Reply:
@miczyg1 this is weird but not my invention. @mkopec ?

Reply:
Looks like these were added in commit https://github.com/Dasharo/docs/commit/778ce53d03542d99db53d6d42f728a60cbfeb852

Reply:
This is what auto-fixer added because the plaintext URLs were used. We should simply always use markdown notation instead.

Reply:
This is this rule: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md034---bare-url-used

Reply:
We can set it to `false` so it's not checked and not corrected. Maybe it could be extended to use markdown syntax instead, when correcting URL. Currently it's not configurable.

Reply:
Switched to md syntax